src/symbol/transform_text.js

```diff
@@ -19,3 +20,16 @@ export default function(text: string, layer: SymbolStyleLayer, feature: Feature)
 
     return text;
 }
+
+
+export default function(text: string | Formatted, layer: SymbolStyleLayer, feature: Feature) {
+    if (text instanceof Formatted) {
+        // OK to transform in place?
```

Comment:
I think it's probably fine... especially since we probably want to move towards having text transforms happen via expressions rather than through `text-transform` anyway